staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -876,11 +901,18 @@ type processorListener struct {
 	resyncLock sync.Mutex
 }
 
-func newProcessListener(handler ResourceEventHandler, requestedResyncPeriod, resyncPeriod time.Duration, now time.Time, bufferSize int) *processorListener {
+// HasSynced returns true if the source informer has synced, and all
+// corresponding events have been delivered.
```

Comment:
```suggestion
// corresponding events have been delivered to every event handler.
```

is that edit correct?

Reply:
oh, this is the lower level one.  nm.